internal/events/service.go

```diff
@@ -51,7 +52,7 @@ type MockEmitter struct {
 	sent []models.Event
 }
 
-func (e *MockEmitter) Send(evt models.Event) error {
+func (e *MockEmitter) Send(_ context.Context, evt models.Event) error {
```

Comment:
Have we considered putting the mock definitions in a test package?

Reply:
Yea. We don't have a set rule for where to put mocks (or even to use a mocking library vs Mock structs).